### ISSUE TITLE
Duplicated return value

### ISSUE THREAD

**Issue by Eneroth3** [SketchUp/api-issue-tracker#532]
In both the Tool interface and the Animation interface, there are methods listing the return value twice. this seems to be a bug in how the docs are built.

![2020-09-16_15h27_05](https://user-images.githubusercontent.com/5631725/93343890-4f8dcb00-f831-11ea-953e-abd63f957c0d.png)

![2020-09-16_15h22_15](https://user-images.githubusercontent.com/5631725/93343913-54eb1580-f831-11ea-9cd9-3c939765e330.png)

![2020-09-16_15h25_04](https://user-images.githubusercontent.com/5631725/93343940-5ae0f680-f831-11ea-8e63-8c6f7594c1e9.png)

**Comment by DanRathbun** [comment]
> ... this seems to be a bug in how the docs are built.

I don't think so. If you look at the doc for [`Tool#enableVCB?`](https://ruby.sketchup.com/Sketchup/Tool.html#enableVCB%3F-instance_method), you'll see that there is no "parroting" happening.

So this is human error. The method docstrings are parroting what the [YARD `@return`](https://rubydoc.info/gems/yard/file/docs/Tags.md#return) tag is saying.

Also there should not be important informational sentences below the code examples. They should be up above the examples with the rest of the _docstring_.

----

Suggested keeping this issue for [`Sketchup::Tools`](https://ruby.sketchup.com/Sketchup/Tool.html) class (and open a separate issue for the [`Sketchup::Animation`](https://ruby.sketchup.com/Sketchup/Animation.html) class.)

**Comment by thomthom** [comment]
Yea, this is some YARD weirdness. I've seen YARD duplicate or add extra return statements. Haven't figured out exactly why and what causes/triggers that.

In this case `onKeyDown` has an explicit `@return` tag, so there should be no `return nil` line in the docs like that.

I wonder if YARD is struggling with `@overload` statements...

**Comment by DanRathbun** [comment]
> Haven't figured out exactly why and what causes/triggers that.

Many of the booboos I noticed in the `Camera` class, have parameter names for `@return` which doesn't say it accepts them.
https://rubydoc.info/gems/yard/file/docs/Tags.md#return
